include/OnnxMlirRuntime.h

```diff
@@ -124,12 +124,17 @@
  * \subsection freeing-tensor-memory Freeing Tensor Memory
  *
  *
- * When creating an OMTensor or OMTensorList with a specified data pointer, a user has the option to
+ * When creating an OMTensor or OMTensorList with a specified tensor array, a user has the option to
  * create a tensor or tensorList with or without ownership (omTensorCreateWithOwnership or omTensorListCreateWithOwnership).
- * If the ownership flag is set to "false", then a user is responsible for freeing memory until the last use.
- * Otherwise, if the flag is set to "true", then destruction of the tensor (omTensorDestroy or omTensorListDestroy)
+ *
+ * In the case of OMTensor, if the ownership flag is set to "false", then a user is responsible for freeing memory until the last use.
+ * Otherwise, if the flag is set to "true", then the destruction of the tensor (omTensorDestroy)
  * will also free any associated memory and data buffers.
  *
+ * With respects to OMTensorList, no matter the value of the ownership flag, every tensor will be destroyed. 
+ * However, if the ownership flag is set to "true", then the tensor data will be destroyed (omTensorListDestroy) and the 
+ * pointer to the tensor list will be freed. Otherwise, if the ownership flag is false, the tensor array is implicitly freed before its last use.
+ *
```

Comment:
* With respects to OMTensorList, no matter the value of the ownership flag, every OMTensor object in the OMTensorList will be destroyed. 
 * If the OMTensorList ownership flag is set to "true", then the array of OMTensor pointers will be destroyed with the OMTensorList. 
  * Otherwise, if the ownership flag is false, the array of OMTensor pointers will not be destroyed with the OMTensorList and must be released separately.